.github/ISSUE_TEMPLATE/integration_request.yaml

```diff
@@ -0,0 +1,78 @@
+name: New Tool Integration
+description: Keptn should integrate with this tool. Let's make it happen...
+title: "[integration] "
+labels: ["integrations"]
+assignees:
+  - agardnerit
```

Comment:
I doubt you want a default assignee in the template. If so, I would rather assign it to a team instead of the individual